### ISSUE TITLE
[Discovery] Learning Center (MVP) Card Sort Research Plan

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#11012]
## User Story or Problem Statement
As a UX Designer, I need to create a research plan for our card sort so I can document the research's purpose, goals, hypothesis, and usability testing recruitment needs. 

Our card sort will help us understand how our proposed audience tags resonate with Veterans, other audiences, and content authors 

[Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/content/tier-2-content-IA-and-design/learning-center-mvp)
[Status, Decisions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/status-and-decisions.md)

## Goal
_A completed research plan sent to Perigean for card sort recruitment_

## Objectives or Key Results this is meant to further
- _User testing to validate the audience tags we are planning for the learning center_

## Tasks
- [x] _Create [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/card-sort-research-plan-veterans.md)_
- [x] _Create [email template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/card-sort/card-sort-recruit-email-veterans.md) for recruiting_
- [x] _Review research plan and email template with product team_
- [x] _Send research plan to Lauren Alexanderson for approval_
- [x] _Send recruiting email text to Perigean_
- [x] _Add research plan to GH_

## Acceptance Criteria
- [ ] Research plan created and uploaded to GH

**Comment by jenniferlee-dsva** [comment]
@andaleliz - Thanks for starting the card sort plan! Great start. Attaching here my feedback and questions. I also included the list of audience and topic tag labels right in the doc: 

[card-sort-research-plan-draft-jl.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5008464/card-sort-research-plan-draft-jl.docx)

The most important clarification for the card sort: 

- With Veterans, we should test with the beneficiary audience labels -- but not mix in the non-beneficiary audience labels. Rational: Veterans and beneficiary audiences will not typically be looking for content that's for non-beneficiaries like school admins and researchers. 

- With content authors - we should test with a mix of beneficiary AND non-beneficiary audience labels, since they may be creating content for both audiences at different times. 

I'll create a separate ticket for creating a list of article titles for the card sort. FYSA @MickinSahni

**Comment by jenniferlee-dsva** [comment]
Hi @MickinSahni - I don't think this should be closed yet bc I have not seen Liz incorporate feedback into the research plan and review with the team yet. 

Will reopen for now. @andaleliz - I know you're focused on LC prototype usability research synthesis, so let's sync on this and other card sort related tickets when you're ready.

**Comment by jenniferlee-dsva** [comment]
@andaleliz - made a few corrections to the og ticket. Lauren's our contact on research, not Andrea; and because this is just recruiting via Perigean, I believe we supply them with the email text (w/ the link to the card sort in optimal) that they send out to Veterans. (Not 100 on details of the email part, so Lauren can advise when you get closer to that point.)

**Comment by andaleliz** [comment]
@jenniferlee-dsva thank you! I had copied and pasted this from another ticket and hadn't cleaned up yet. I'll review the doc you uploaded and focus on the card sort later this week, once synthesis is complete for the usability study.

**Comment by andaleliz** [comment]
>With Veterans, we should test with the beneficiary audience labels -- but not mix in the non-beneficiary audience labels. Rational: Veterans and beneficiary audiences will not typically be looking for content that's for non-beneficiaries like school admins and researchers.

I agree that it's not typical, but there were people in our study who are both Veterans, and people who work with Veterans.  They were interested in the links in that audience box. We can leave it out if you feel strongly about it, but wanted to raise this point.

**Comment by andaleliz** [comment]
@jenniferlee-dsva have you seen the [email template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/card-sort-recruit-email.md) I put together for Perigean? Interested to know if you have any feedback.

**Comment by andaleliz** [comment]
@jenniferlee-dsva following up with you on the email template - do you have any edits? I've updated the research plan to include your feedback, with the exception of adding the topics and audiences to the resources section. I'll do that once it's all finalized in #12329 to minimize the number of places we need to make updates.

The plan currently has the week of 9/7 as our target date for the email to be sent out, taking into account the week of lead time requested between submitting a research plan and kicking it off.  Does that sound reasonable to you? That means we'd need to get everything finalized and submitted by 8/31.

RE couple of comments you made in the research plan:
>Since we’re going to allow authors to tag each article with up to 3 audience labels, we may want to allow this for participants as well – they don’t have to, but tell them that they can? 

>It’s still TBD how many total topic tags should be allowed, so for the card sort, perhaps we can allow as many as the user wants? This might tell us how users and authors might understand these labels – and potentially what governance we need to build into CMS.

OptimalSort only allows people to add one article to one category (tag) at a time, so our card sort will focus on how people understand the **primary** audience/topic categories for articles. If it's a priority to evaluate the application of multiple labels at one time, then we should look at other tools/methods to get this feedback from people.

**Comment by jenniferlee-dsva** [comment]
Replying in reverse order: 
>OptimalSort only allows people to add one article to one category (tag) at a time, so our card sort will focus on how people understand the primary audience/topic categories for articles.

Sounds good. Simpler. 

>... 9/7 as our target date for the email to be sent out...That means we'd need to get everything finalized and submitted by 8/31.

Yes, that sounds reasonable and I think we can def hit that 8/31 target. 

>Email template

I submitted MD edits in a branch. You should have it in your review/assigned queue. Per my PR comments, please be sure to vet the final email text with @lalexanderson-dsva and @JeffBarnesUSDS to make sure we're not veering into PRA violation territory. Thank you @andaleliz

**Comment by andaleliz** [comment]
@jenniferlee-dsva I'm not seeing any active branches from you, or anything in my review/assigned queue. Can you double check that I'm assigned? 

I think Lauren will review email template as part of the review process of the research plan but will also check w/ Jeff too.

**Comment by andaleliz** [comment]
Submitted to Lauren this AM for her review.

Also, FYI to all, moved our card sort docs/files into [a dedicated folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/discovery-and-research/card-sort) in our `discovery-and-research` directory. Things were getting messy and will only get worse once we add synthesis docs.

**Comment by andaleliz** [comment]
Lauren submitted this to Perigean today! Closing out.
